themes/custom/az_barrio/az_barrio.theme

```diff
@@ -388,7 +388,7 @@ function az_barrio_primary_logo() {
   $primary_logo_path_parts = pathinfo($primary_logo_path['path']);
   $primary_alt_text = Html::escape(theme_get_setting('primary_logo_alt_text'));
   $primary_title_text = Html::escape(theme_get_setting('primary_logo_title_text'));
```

Comment:
Escaping the alt and title text here.

Reply:
These may not be necessary if this alt and title text variables are only used in render arrays.